.pre-commit-config.yaml

```diff
@@ -14,7 +14,7 @@ repos:
     hooks:
       - id: go-mod-tidy-repo
       - id: go-test-repo-mod
-        args: [ -race ]
+        args: [ -race, -count=1 ]
```

Comment:
Can you also add it to the [vscode test config](https://github.com/caas-team/sparrow/blob/main/.vscode/settings.json#L2)?